From foo@baz Mon Oct 26 16:43:54 PDT 2009
Date: Mon, 26 Oct 2009 16:43:54 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: dream: mark as BROKEN

It doesn't build, and hasn't for a long time (if ever).  So mark
it BROKEN for now.

Cc: Pavel Machek <pavel@ucw.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/dream/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/dream/Kconfig
+++ b/drivers/staging/dream/Kconfig
@@ -1,5 +1,6 @@
 config DREAM
        tristate "HTC Dream support"
+       depends on BROKEN
 
 source "drivers/staging/dream/smd/Kconfig"
 
